src/Exception/InvalidMutatorException.php

```diff
@@ -9,7 +9,7 @@
 
 namespace Infection\Exception;
 
-use Infection\Mutator\Util\Mutator;
```

Comment:
any reason behind it?

Reply:
It seems like the logic behind this is that this is a major class/interface that deserves to be at the root of the similarly named namespace.

Reply:
> deserves to be at the root of the similarly named namespace

sounds reasonable

> Maybe moving them all one directory level down would be a better bet?

I'm fine with it, the only one note here is to update these lines

https://github.com/infection/infection/blob/ef45b8e6f6eec00821b385ba670a4ca95c2a9d20/tests/Mutator/Util/MutatorProfileTest.php#L42-L43

probably, it's out of the scope of this PR